src/services/database.ts

```diff
@@ -0,0 +1,56 @@
+import mysql from 'mysql';
```

Comment:
Services usually contains Business Logic and not Direct Database Interactions, Database interactions could go into a domain/interaction Layer, I have found a small article related to this: [3 Layer with Node](https://bytearcher.com/articles/node-project-structure/)